functions/demo/controllers/DemoController.ts

```diff
@@ -105,6 +106,24 @@ export class DemoController {
     );
   }
 
+  private async executeSetupPhenotype(req: Request, res: Response) {
+    const steps: IStepTask[] = [
+      {
+        code: "phenotype",
+        message: "Running Phenotype flow...",
```

Comment:
Suggest to use sentence case:
```suggestion
        message: "Running phenotype flow...",
```